backend/games/wordGuesser/ClassicWordle.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+
+class ClassicWordle:
+    # maximum number of guesses
+    MAX_GUESSES = 6
+    # represents the state of the game, index 0 will hold the "goal word"
+    gameBoard = List[str]
+
+    # clears any previous activity and randomizes a word to initialize the list
+    def startGame(self) -> List[str]:
+        pass
+
+    # clears all previous progress and calls checkWin() to determine final results
+    # will return a victory or failure string
+    def endGame(self) -> str:
+        pass
```

Comment:
If `startGame()` already clears the game state, why does `endGame()` need to?

And why have an `endGame()` at all? Could the win/loss state be returned with `trackProgress()`?

Reply:
My original idea was for start_game() to clear previous game data as a precaution, such that starting a new game will not be affected by any previous game data. Ideally, end_game() would be called within start_game().

having this as a separate function also allows the users to terminate the game early if they decide to quit and find out what the word was, all without starting a new game

Reply:
That makes sense!